src/main/menu.js

```diff
@@ -147,6 +147,10 @@ export function setupMenu(store) {
   const archives = getAllArchives(state);
   const currentArchiveId = getCurrentArchiveId(state);
   let condenced = Boolean(getSetting(state, 'condencedSidebar'));
+  let menubarVisibility = getSetting(state, 'menubarVisibility');
```

Comment:
Would naming this `menubarVisible` make more sense? Just because of how it's used..